app/assets/stylesheets/content/_datepicker.sass

```diff
@@ -0,0 +1,202 @@
+//-- copyright
+// OpenProject is a project management system.
+// Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License version 3.
+//
+// OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+// Copyright (C) 2006-2013 Jean-Philippe Lang
+// Copyright (C) 2010-2013 the ChiliProject Team
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+//
+// See doc/COPYRIGHT.rdoc for more details.
+//++
+
+@import jquery_ui
+
+//$dp- var prefix for date picker
+
+$dp-theme-color: #CCCCCC
+$dp-theme-lines-color1: #FFFFFF
+$dp-theme-lines-color2: #F8F8F8
+$dp-theme-border-color: #DDDDDD
+
+$dp-header-elements-height: 30px
+$dp-header-top: 5px
+$dp-header-padding: .2em 0
+
+$dp-hover-normalisation: 2px
+$dp-round-corners-radius: 0
+$dp-cell-corner-radius: 3px
+$dp-day-margin: 2px
+$dp-day-margin-hover: 1px
+$dp-days-text-color: #535353
+
+$dp-week-padding-top: 7px
+$dp-week-padding-bottom: 7px
+$dp-week-text-color: #CCCCCC
+
+$dp-border: 1px solid $dp-theme-border-color
+
+$dp-fon-size-rewrite: .9em
+
+$dp-number-of-columns: 8
+$dp-table-width: 100%
+
+$dp-month-year-width: 40%
+$dp-month-year-margin: 0 4.5%
+
+.noselect
```

Comment:
@vcostin consider using [placeholder selectors](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#placeholder_selectors_) rather than concrete classes for such use cases